README.md

```diff
@@ -50,26 +61,41 @@ The `GameBoard` class is the core of the game logic and user interaction. It con
 - **`__init__(self, size)`**: Initializes the game board with a specified size and sets up the initial grid with empty cells.
```

Comment:
check the __init__ function because now it is a littel bit different... for shure have many more attributes + the size of the grid now is fix to 4